lib/DBIx/Class/Manual/Joining.pod

```diff
@@ -14,16 +14,15 @@ If you don't know the basics of DBIx::Class, read
 L<DBIx::Class::Manual::Intro> first. Skip this section if you know how
 SQL joins work.
 
-But I'll explain anyway. Assuming you have created your database in a
-more or less sensible way, you will end up with several tables that
-contain C<related> information. For example, you may have a table
-containing information about C<CD>s, containing the CD title and its
-year of publication, and another table containing all the C<Track>s
-for the CDs, one track per row.
+If you have created your database sensibly, you will have several tables
+that contain C<related> information. For example, you may have a table
+representing C<CD>s, containing each CD's title and its year of
+publication, and another table representing all the C<Track>s for the
+CDs, one track per row.
 
-When you wish to extract information about a particular CD and all
-its tracks, You can either fetch the CD row, then make another query
-to fetch the tracks, or you can use a join. Compare:
+To retrieve information about a particular CD and all its tracks, either
+fetch the CD row, then make another query to fetch the tracks, or use a
+join. Compare:
```

Comment:
> How about I link to those instead?

Excellent! I'd order the visual explainer first. Also since you are in this headspace already: could you possibly work a sentence about this somewhere: https://rt.cpan.org/Ticket/Display.html?id=104289#txn-1494725. I am not trying to create more work for you, this is just tightly related and has been gathering dust for a while...

Cheers!